backend/app/schemas/menu.py

```diff
@@ -10,11 +10,12 @@
 
 class MenuBase(SchemaBase):
     name: str
-    parent_id: int = Field(default=None, ge=1, description='菜单父级ID')
+    parent_id: int | None = Field(default=None, description='菜单父级ID')
     sort: int = Field(default=0, ge=0, description='排序')
     icon: str | None = None
     path: str | None = None
     menu_type: MenuType = Field(default=MenuType.directory, ge=0, description='菜单类型（0目录 1菜单 2按钮）')
+    meta: dict | None = None
```

Comment:
There seems to be a discrepancy between this and the front-end, where the front-end meta contains a variety of information, including icons, etc. The back-end separates them, and I think we should update the front-end instead of adding an unknown meta column